source/developers/security-token-service/AssumeRoleWithWebIdentity.rst

```diff
@@ -117,6 +119,16 @@ This endpoint supports the following query parameters:
        See :ref:`minio-access-management` for more information on MinIO
        authentication and authorization.
 
+   * - ``RoleARN``
+     - string
+     - *Optional*   
+
+       The role ARN to use for all user authentication requests.
+       If used, there must be a Role Policy defined for the provider by the ``role_policy`` configuration parameter or the ``MINIO_IDENTITY_OPENID_ROLE_POLICY`` environment variable.
+       Likewise, if you configure a Role Policy, you must also specify the ``RoleARN``.
```

Comment:
It's somewhat more nuanced:

The RoleARN must match that of the OIDC configuration you want to use for this claim.

There must be a matching OIDC RolePolicy-based configuration on MinIO for this to work.

If you do *not* specify RoleARN, MinIO assumes you are attempting a JWT-based claim (I believe).